src/main/resources/assets/configured/lang/pl_pl.json

```diff
@@ -1,27 +1,27 @@
 {
     "configured.gui.search": "Szukaj...",
-    "configured.gui.reset_all": "Reset All",
+    "configured.gui.reset_all": "Resetuj całe",
     "configured.gui.edit": "Edytuj",
     "configured.gui.edit_value": "Edycja wartości",
     "configured.gui.add_value": "Dodaj...",
     "configured.gui.reset": "Resetuj",
     "configured.gui.remove": "Usuń",
-    "configured.gui.modify": "Modify",
-    "configured.gui.restore": "Restore",
-    "configured.gui.select": "Select",
-    "configured.gui.change": "Change",
-    "configured.gui.save": "Save",
-    "configured.gui.select_world": "Pick World",
-    "configured.gui.no_permission": "§cYou do not have permissions to do this",
-    "configured.gui.not_installed": "§cConfigured is not installed on the server",
-    "configured.gui.edit_world_config": "Select World To Edit §e%s§r Config",
```

Comment:
One more question, is this dynamically inserted text string really needed? From my observations, it has a few issues:
* the text is supposed to be yellow, but it is not
* the text is always in English
* it seems to be used for server-side configs only, so the word "server" could be there right from the beginning instead of being inserted dynamically.

Reply:
Realistically I should be colouring them in code. The "server" label is actually the name of the config since it's possible to have multiple servers configs from a single mod.

Reply:
Thank you for the clarification. Since this line is now translated as follows: "Select a world to edit the config of type %s", I'll definitely have to modify it a bit.